website/snippets/_sl-partner-links.md

```diff
@@ -28,6 +28,20 @@ The following tools integrate with the dbt Semantic Layer:
     </a>
   </div>
 
+<div className="card-container">
+  <Card
+    title="KlipFolio Power Metrics"
+    body="Refer to KlipFolio's documentation to learn how to connect, and deliver a streamlined metrics catalog to your business users."
+    link="https://support.klipfolio.com/hc/en-us/articles/18164546900759-PowerMetrics-Adding-dbt-Semantic-Layer-metrics"
+    icon="klipfolio"/>
```

Comment:
The dark mode icon is hard to see (black icon on dark background). Not a showstopper but might not be ideal from a branding perspective.

Reply:
totally agree and i played around it already but i couldn't do much with that unfortunately. this is because they only have [black or white](https://www.klipfolio.com/branding-guidelines/klipfolio-badge) so it would either be difficult in light mode or dark mode. the other option is just black which also isn't great

Reply:
the other icons from their branding page were too small so went with badges